src/NServiceBus.Gateway/Config/ChannelCollection.cs

```diff
@@ -1,4 +1,4 @@
-#if NET452
+#if NET472
```

Comment:
isn't this XML based configuration that is supposed to be removed with this major version?